skbeam/core/roi.py

```diff
@@ -398,8 +398,54 @@ def mean_intensity(images, labeled_array, index=None):
     return mean_intensity, index
 
 
-def circular_average(image, calibrated_center, threshold=0, nx=100,
-                     pixel_size=(1, 1),  min_x=None, max_x=None):
+
+def bin_1D(x, y, nx=None, min_x=None, max_x=None):
```

Comment:
@yugangzhang Can you please take out the bin_1D function and use the same function from skbeam.core.utils.?